crates/rethnet_rpc_server/src/lib.rs

```diff
@@ -894,10 +919,20 @@ impl Server {
                     network_id,
                 })
             } else {
+                let rethnet_state = HybridState::with_accounts(genesis_accounts);
+                let blockchain = Arc::new(RwLock::new(LocalBlockchain::new(
+                    &rethnet_state,
+                    spec_id,
+                    config.gas,
+                    config.initial_date,
+                    Some(RandomHashGenerator::with_seed("seed").next_value()),
+                    config.initial_base_fee_per_gas,
+                )?));
+                let rethnet_state = Arc::new(RwLock::new(Box::new(rethnet_state)));
```

Comment:
Same as above

Reply:
This I just copied from the existing usage just above, and it caught my eye too, but it did just work and then I forgot to circle back to it, so thank you for pointing it out. Removed in https://github.com/NomicFoundation/hardhat/pull/4221/commits/048a33be02b7c3f84bad1528522050d7699ca6ef